### PR TITLE
Add documents to the flask charm

### PR DESCRIPTION
**Pull request by weiiwang01**
Include an overall description, a beginner's tutorial, and a contribution guide in the Flask charm.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+## Project and community
+
+The Flask Operator is a member of the Ubuntu family. It's an open source
+project that warmly welcomes community projects, contributions, suggestions,
+fixes and constructive feedback.
+
+- [Code of conduct](https://ubuntu.com/community/code-of-conduct)
+- [Get support](https://discourse.charmhub.io/)
+- [Join our online chat](https://chat.charmhub.io/charmhub/channels/charm-dev)
+- [Contribute](Contribute)
```
Comment (jdkandersson) [inline README.md]: Does this link go anywhere?

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+## Project and community
+
+The Flask Operator is a member of the Ubuntu family. It's an open source
+project that warmly welcomes community projects, contributions, suggestions,
+fixes and constructive feedback.
+
+- [Code of conduct](https://ubuntu.com/community/code-of-conduct)
+- [Get support](https://discourse.charmhub.io/)
+- [Join our online chat](https://chat.charmhub.io/charmhub/channels/charm-dev)
+- [Contribute](Contribute)
```
Comment (jdkandersson) [inline docs/index.md]: Does this link to anywhere?

File: docs/tutorial/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+your own container' mode, meaning it doesn't provide a workload container. A
+container image for use with the flask-k8s charm must meet the following
+requirements:
+
+- It should have gunicorn installed as a global Python package
+- The Flask application should be situated in the `/srv/flask/app` directory
+- The Flask WSGI application should be located at `/srv/flask/app/app.py`, with
+  the variable name `app`
+- The Flask application must invoke `app.config.from_prefixed_env()` to receive
+  charm configuration
```
Comment (jdkandersson) [inline docs/tutorial/getting-started.md]: Perhaps remove charm and just say "to receive configuration"?

File: README.md (hunk 1 above)
Comment (weiiwang01) [reply README.md]: Fixed, thanks!

File: docs/index.md (hunk 2 above)
Comment (weiiwang01) [reply docs/index.md]: Fixed, thanks!

File: docs/tutorial/getting-started.md (hunk 3 above)
Comment (weiiwang01) [reply docs/tutorial/getting-started.md]: Fixed, thanks!

**Review by jdkandersson — DISMISSED** [review]

File: docs/how-to/contribute.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+## Developing
+
+### Building from Source
+
+To build and deploy the flask-k8s charm from source, follow the steps below.
+
+#### Docker Image Build
+
+Build your Flask application image, and to allow microk8s to pick up the locally
+built image, you must export the image and import it within microk8s.
```
Comment (yanksyoon) [inline docs/how-to/contribute.md:36]: Minor nitpick, I'm wondering if adding a `docker save image:tag > image.tar && microk8s import image.tar` would help :)

**Review by yanksyoon — DISMISSED** [review]
LGTM!

**Review by merkata — DISMISSED** [review]

**Review by arturo-seijas — DISMISSED** [review]

File: docs/how-to/contribute.md (hunk 4 above)
Comment (weiiwang01) [reply docs/how-to/contribute.md:36]: Added, thanks!

**Review by merkata — APPROVED** [review]

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Pull request merged**